### ISSUE TITLE
[GITADORA]Known issue : new card register is not working in NEX+AGE

### ISSUE THREAD

**Issue by JamesLewisLiu** [asphyxia-core/plugins#36]
EXCHAIN is working, or play other games first.

**Comment by thomeval** [comment]
This seems to work fine with Guitar Freaks Nextage and the current version (v1.2.3) of the Gitadora plugin. Is this still an issue?

**Comment by JamesLewisLiu** [comment]
> This seems to work fine with Guitar Freaks Nextage and the current version (v1.2.3) of the Gitadora plugin. Is this still an issue?

Yeah it still says something like "NETWORK is BUSY" when you "swipe" a complete new card 
And this time Asphyxia's log window says   [gitadora@asphyxia] error: [profiles] Request data is missing required parameter: player.refid
![image](https://user-images.githubusercontent.com/17558260/168478183-756e6030-a7dd-4c5f-bd1a-fe838c225b61.png)
